samples/sampler/stories/qa/Scroller.js

```diff
@@ -226,6 +227,14 @@ export const EditableList = (args) => {
 		setItems(itemsArr);
 	}, [dataSize]);
 
+	const onClickRemoveButton = useCallback((ev) => {
+		if (removeItem.current) {
+			removeItem.current();
+		}
+		ev.preventDefault();
+		ev.stopPropagation();
+	}, [removeItem]);
```

Comment:
Would you remove `removeItem` from dependency array?

Reply:
done